central/blob/datastore/search/searcher_impl.go

```diff
@@ -0,0 +1,62 @@
+package search
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/blob/datastore/index"
+	"github.com/stackrox/rox/central/blob/datastore/store"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/blevesearch"
+	"github.com/stackrox/rox/pkg/search/paginated"
+)
+
+type searcherImpl struct {
+	storage           store.Store
+	indexer           index.Indexer
+	formattedSearcher search.Searcher
+}
+
+func (s *searcherImpl) SearchIDs(ctx context.Context, q *v1.Query) ([]string, error) {
+	var (
```

Comment:
No need to declare these above? Instead just declare them with the := on first use